ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2538,6 +2538,27 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-sno
+- as: gcp-ipi-workers-marketplace-f14
+  cron: 13 20 12,27 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @network-ovnkubernetes'
+      TAG_VERSION: '@4.14'
```

Comment:
Suggest to set `COMPUTE_OSIMAGE` here too, so that in the future, we can easily switch over the image by versions.